src/lumigo_tracer/wrappers/http/http_data_classes.py

```diff
@@ -8,13 +8,15 @@ class HttpRequest:
     uri: str
     headers: dict
     body: bytes
+    instance_id: bytes
 
     def __init__(self, **kwargs):
         self.host = kwargs["host"]
         self.method = kwargs["method"]
         self.uri = kwargs["uri"]
         self.headers = {k.lower(): v for k, v in (kwargs.get("headers") or {}).items()}
         self.body = kwargs.get("body")
+        self.instance_id = kwargs.get("instance_id")
```

Comment:
self.instance_id maybe should be `Optional[bytes]`?

Reply:
Is it the same as 'body'?
I don't see any optional

Reply:
You right, change body too :)